courses/fundamentals_of_ada/quiz/limited_private/quiz.rst

```diff
@@ -0,0 +1,28 @@
+.. code:: Ada
+
+    package P is
+       type Priv is private;
+    private
+       type Lim is limited null record;
+    end P;
+
+Which of the following piece(s) of code is(are) legal?
```

Comment:
This is a little confusing. Put a comment or something after "type Lim" to show where the answers would actually appear in the package